lochness/rpms/__init__.py

```diff
@@ -57,8 +59,7 @@ def initialize_metadata(Lochness: 'Lochness object',
                                 str.
         multistudy: True if the rpms repo contains more than one study's data
     '''
-    study_rpms = Lochness['keyring'][f'rpms.{study_name}']
-    rpms_root_path = study_rpms['RPMS_PATH']
+    rpms_root_path = Lochness['RPMS_PATH']
```

Comment:
It appears that previously it was:

```cfg
rpms:
  study_name:
    RPMS_PATH:
mediaflux:


xnat:

    
```

Now, has it been made:
```cfg
RPMS_PATH:
rpms:
  study_name:
  
mediaflux:


xnat:


```

Reply:
Yes, I've moved it out of the study, since all studies will have the same RPMS_PATH (RPMS server has been set that way for us by Larry in Prescient network).